tests/Integration/Behaviour/Features/Context/Domain/ModuleFeatureContext.php

```diff
@@ -107,4 +118,20 @@ public function updateModuleStatus(string $action, string $technicalName): void
         // Clean the cache
         Module::resetStaticCache();
     }
+
+    /**
+    * @When /^I bulk uninstall modules: "(.+)"$/
+    */
+   public function bulkUninstallModule(string $modulesRef): void
+   {
+       $modules = [];
+       foreach (PrimitiveUtils::castStringArrayIntoArray($modulesRef) as $modulesReference) {
+           $modules[] = $modulesReference;
+       }
+
+       $this->getQueryBus()->handle(new BulkUninstallModuleCommand($modules));
+
+       // Clean the cache
+       Module::resetStaticCache();
```

Comment:
The clear cache was useful when we used legacy code, I believe this won't be useful anymore with the CQRS implementation But we can keep it for now and check later it we could get rid of it